web-deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This file deploys my website.
```

Comment:
Maybe move the install of nginx to this script instead of local-setup.sh?

Reply:
Done and tested.